fuzz/src/lib.rs

```diff
@@ -45,7 +45,9 @@ pub trait DbSimulator {
 
 	fn map_operation(operation: &Self::Operation) -> parity_db::Operation<Vec<u8>, Vec<u8>>;
 
-	fn model_content(model: &Self::Model) -> Vec<(Vec<u8>, Vec<u8>)>;
+	fn model_required_content(model: &Self::Model) -> Vec<(Vec<u8>, Vec<u8>)>;
+
+	fn model_optional_content(model: &Self::Model) -> Vec<(Vec<u8>, Vec<u8>)>;
```

Comment:
Yes, definitely.  But it starts getting very quickly annoying because we are in a trait so `impl` return types don't work. It's why I have been lazy and I have not bothered to do it. It's likely to don't matter much anyway, we are opening/closing databases a lot during fuzzing so some extra allocations are likely to don't have any significant impact. Is it fine for you?

Reply:
definitely fine :)